src/app/sponsors/sponsors.component.html

```diff
@@ -25,15 +25,4 @@ <h5>Workflow Languages</h5>
       </div>
     </div>
   </div>
-  <div *ngIf="!(user$ | async) && isHomePage" class="row text-center">
-    <h5>Works With</h5>
-    <br />
-    <div fxLayout="row wrap" fxLayoutAlign="space-around stretch">
-      <div *ngFor="let partner of partners" class="pb-5 px-1">
```

Comment:
only reference of partners removed?  I feel like there's some clean up that needs to be done